modules/core/ColorSystem.js

```diff
@@ -0,0 +1,33 @@
+import { AbstractSystem } from './AbstractSystem';
+import colorsJSON from '../../data/colors.json';
```

Comment:
The color system should eventually depend on the data loader for all its data needs- instead of importing the colors.js like this.  Reason: eventually, we might want to let the user load their OWN color information, which they can't do via imports like this.